package.json

```diff
@@ -26,19 +29,41 @@
   ],
   "license": "MIT",
   "devDependencies": {
+    "@commitlint/config-conventional": "^8.3.4",
+    "@mixmaxhq/commitlint-jenkins": "^1.4.2",
     "@mixmaxhq/prettier-config": "^1.0.0",
+    "@mixmaxhq/semantic-release-config": "^2.0.0",
+    "cz-conventional-changelog": "^3.1.0",
     "eslint": "^6.7.2",
     "eslint-config-mixmax": "^4.2.0",
     "express": "^4.15.2",
     "jasmine": "^2.5.3",
     "jasmine-promises": "github:mixmaxhq/jasmine-promises#working_in_node",
-    "prettier": "^1.19.1"
+    "prettier": "^1.19.1",
+    "semantic-release": "^17.0.1"
   },
   "dependencies": {
     "object.getownpropertydescriptors": "2.0.3"
   },
   "repository": {
     "type": "git",
     "url": "git+ssh://github.com/mixmaxhq/promise-callbacks.git"
+  },
+  "config": {
+    "commitizen": {
+      "path": "./node_modules/cz-conventional-changelog"
+    },
+    "manualPublishMessage": "This repository is configured to use semantic-release for its releases. Please do not release manually.\n"
+  },
+  "commitlint": {
+    "extends": [
+      "@commitlint/config-conventional"
+    ]
+  },
+  "release": {
+    "extends": "@mixmaxhq/semantic-release-config"
+  },
+  "publishConfig": {
+    "access": "public"
```

Comment:
Is this necessary given that the package isn't scoped?

Reply:
Probably not; I do think it's somewhat nice to be explicit here. (this was also automatically added with the `mixmax dev repo add-semantic-release` script)